src/main/resources/views/blocks/topic/data.ftl

```diff
@@ -105,6 +106,42 @@
                     </div>
                 </div>
             </li>
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle"
+                   href="#"
+                   role="button"
+                   data-toggle="dropdown"
+                   aria-haspopup="true"
+                   aria-expanded="false">
+                    <strong>After offset:</strong> ()
+                </a>
+                <div class="dropdown-menu khq-offset-navbar">
+                    <div class="input-group">
+                        <table>
+                            <#list 0..partitions-1 as partition>
+                                <tr>
+                                    <td class="offset-navbar-partition-label offset-navbar-partition-td">Partition ${partition} : </td>
+                                    <td class="offset-navbar-partition-td">
+                                        <input class="form-control"
+                                               type="number"
+                                               min="0"
+                                           name="${partition}"
+                                           type="text"
+                                            <#if navbar["offset"][partition?string]??>
+                                                value="${navbar["offset"][partition?string]}"
+                                            </#if>
+                                        />
+                                    </td>
+                                </tr>
+                            </#list>
+
+                        </table>
+                        <div class="input-group-append">
```

Comment:
This button is missing 1 breakpoint : 
![image](https://user-images.githubusercontent.com/2064609/67602895-e95f0c00-f777-11e9-8707-c4611d67df58.png)